src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/masonry-decorator/masonry-view.js

```diff
@@ -0,0 +1,414 @@
+            });
+            this.sandbox.dom.append(this.$el, $empty);
+
+            // render masonry-grid
+            var $grid = this.sandbox.dom.createElement('<div id="' + constants.masonryGridId + '"/>');
+            this.sandbox.dom.append(this.$el, $grid);
+
+            this.sandbox.dom.append($container, this.$el);
+        },
+
+        /**
+         * Show the empty-list-indicator if datagrid contains no data, else hide it
+         */
+        updateEmptyIndicatorVisibility: function() {
+            if (!!this.datagrid.data.embedded && this.datagrid.data.embedded.length > 0) {
+                this.sandbox.dom.hide('.' + constants.emptyIndicatorClass);
+            } else {
+                this.sandbox.dom.show('.' + constants.emptyIndicatorClass);
+            }
+        },
+
+        /**
+         * Start the masonry-plugin on the masonry-grid element
+         */
+        initializeMasonryGrid: function() {
+            this.sandbox.masonry.initialize('#' + constants.masonryGridId, {
+                align: 'left',
+                direction: 'left',
+                itemWidth: 190,
+                offset: 30,
+                verticalOffset: 20
+            });
+        },
+
+        /**
+         * Bind dom related events for datagrid-view
+         */
+        bindGeneralDomEvents: function() {
+            if (this.options.unselectOnBackgroundClick) {
+                this.sandbox.dom.on('.body', 'click.contact.list', function() {
+                    this.deselectAllRecords();
+                }.bind(this));
+            }
+        },
+
+        /**
+         * Parses the data and passes it item by item to a render function
+         * @param records {Array} array with records to render
+         */
+        renderRecords: function(records) {
+            this.updateEmptyIndicatorVisibility();
+            this.sandbox.util.foreach(records, function(record) {
+                var item = processContentFilters.call(this, record);
+                var id, image, title, description;
+
+                id = item.id;
+                image = item[this.options.fields.image].url || '';
+                title = concatRecordColumns(item, this.options.fields.title, this.options.separators.title);
+                description = concatRecordColumns(item, this.options.fields.description, this.options.separators.description);
+
+                // pass the found data to a render method
+                this.renderItem(id, image, title, description);
+            }.bind(this));
+        },
+
+        /**
+         * Renders an masonry-grid item with the given properties
+         * @param id
+         * @param image
+         * @param title
+         * @param description
+         */
+        renderItem: function(id, image, title, description) {
+            this.$items[id] = this.sandbox.dom.createElement(
+                this.sandbox.util.template(templates.item)({
```

Comment:
@nnatter intead of ``util.template(templates.item)(options)`` you can write ``util.template(templates.item, options)``